package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@segment/integration-intercom",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
This is technically a breaking change in that we need to run migrations and the changes are not simply isolated in the integration code.